packages/amplify-graphql-schema-generator/package.json

```diff
@@ -22,7 +22,7 @@
     "access": "public"
   },
   "scripts": {
-    "build": "tsc && cd vpc-db-lambda && rm -rf node_modules && npm install && tsc && cp -r node_modules lib && cd lib && bestzip --force node ../../lib/rds-schema-inspector.zip ./*",
+    "build": "tsc",
```

Comment:
any reason to get rid if this?

Reply:
Oh shoot, I missed that, same as the comment on the package.json for model-transformer, I was developing on win machine earlier, and that build step failed. I didn't revert when I flipped over here.